lib/fastlane/plugin/wpmreleasetoolkit/actions/common/prototype_build_details_comment_action.rb

```diff
@@ -0,0 +1,130 @@
+module Fastlane
+  module Actions
+    class PrototypeBuildDetailsCommentAction < Action
+      def self.run(params)
+        # Get Input Parameters
+        appcenter_org_name = params[:appcenter_org_name]
+        appcenter_app_name = params[:appcenter_app_name]
+        appcenter_release_id = params[:appcenter_release_id]
+        commit = params[:commit] || other_action.last_git_commit[:abbreviated_commit_hash]
+
+        metadata = params[:metadata] # e.g. {'Build Config':… , 'Version': …, 'Short Version': …}
+        direct_link = params[:download_url]
+        unless direct_link.nil?
+          metadata['Direct Link'] = "<a href='#{direct_link}'><tt>#{File.basename(direct_link)}</tt></a>"
+        end
+
+        # Build the comment parts
+        install_url = "https://install.appcenter.ms/orgs/#{appcenter_org_name}/apps/#{appcenter_app_name}/releases/#{appcenter_release_id}"
+        qr_code_url = "https://chart.googleapis.com/chart?chs=500x500&cht=qr&chl=#{CGI.escape(install_url)}&choe=UTF-8"
+        metadata_rows = metadata.compact.map do |key, value|
+          "<tr><td><b>#{key}</b></td><td>#{value}</td></tr>"
+        end
+
+        intro = "📲 You can test the changes from this Pull Request in <strong>#{appcenter_app_name}</strong> by scanning the QR code below to install the corresponding build via App Center."
+        body = <<~COMMENT_BODY
+          <table>
+          <tr>
+            <td rowspan='#{metadata.count + 3}'><img src='#{qr_code_url}' width='250' height='250' /></td>
+            <td width='150px'><b>App Name</b></td><td><tt>#{appcenter_app_name}</tt></td>
```

Comment:
[`<tt>` has been depracted](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt).

What do you think of using `<pre>`?

Reply:
Seems like `<pre>` is a `display:block` element (like a `div`) aimed to stylize an entire block of potentially-multiline text (e.g. ASCII art), while here we might prefer to keep an `display:inline` element (like a `span`) that can be applied to only a portion of the paragraph. So I think `<code>` would be a more appropriate replacement for that `<tt>`

Reply:
Addressed in 911d71bb02533ba7cc618e9f7f1490789d4b6f30

Reply:
Ah, nice! HTML is cool like that.